master/custom/factories.py

```diff
@@ -557,6 +557,15 @@ class Windows64Build(BaseWindowsBuild):
     factory_tags = ["win64"]
 
 
+class Windows64BigmemBuild(BaseWindowsBuild):
+    buildersuffix = ".bigmem"
```

Comment:
What's that for?

Reply:
This is for disambiguation in case there are multiple similar builders on a worker.  The working directory is constructed something like `<local worker workdir prefix>/<branchname>.<workername><buildersuffix>`.  For example, on my Gentoo bot I have `/buildbot/buildarea/cpython/3.x.ware-gentoo-x86.nondebug/` and `/buildbot/buildarea/cpython/3.x.ware-gentoo-x86.installed/`.